db/migrate/20160922023739_add_unique_constraint_to_memberships.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueConstraintToMemberships < ActiveRecord::Migration
+  def change
+    add_index :memberships, [:group_id, :user_id], unique: true
```

Comment:
We should check in prod and make sure we don't have any memberships which are violating this constraint currently.